### PR TITLE
Page duplicate dialog: title field

### PR DESCRIPTION
**Pull request by distantnative**
## Describe the PR

This was mindblowingly easy with the new dialogs implementation

### Enhancements
- Page duplicate dialog features field for directly changing the title

## Breaking changes

None.

## Related issues/ideas

- Implements parts of https://kirby.nolt.io/117

## Ready?

- [x] Unit tests for fixed bug/feature
- [x] In-code documentation (wherever needed)
- [x] CI checks pass

## When merging

- [x] Add to [website docs release checklist](https://github.com/getkirby/getkirby.com/pulls) (if needed)
- [x] Add changes to release notes draft in Notion

### CODE REVIEW

**Comment by distantnative** [comment]
@bastianallgeier do you have an idea why that unit test is failing?

**Comment by afbora** [comment]
I've fixed the `@covers` typo but now this failing about current implementation.

**Comment by distantnative** [comment]
🙈🙈

**Comment by distantnative** [comment]
@bastianallgeier I think this is ready except the failing unit test - which I don't really understand

**Comment by afbora** [comment]
The last issue was about `The title must not be empty` error and I've fixed the CI test.

**Comment by afbora** [comment]
I think ready for review and merge 🚀

**Review by afbora — APPROVED** [review]

File: config/areas/site/dialogs.php [hunk 1]
```diff
@@ -385,14 +385,22 @@
     // duplicate page
     'pages/(:any)/duplicate' => [
         'load' => function (string $id) {
-            $page        = Find::page($id);
-            $hasChildren = $page->hasChildren();
-            $hasFiles    = $page->hasFiles();
+            $page            = Find::page($id);
+            $hasChildren     = $page->hasChildren();
+            $hasFiles        = $page->hasFiles();
+            $toggleWidth     = '1/' . count(array_filter([$hasChildren, $hasFiles]));
```
Comment (bastianallgeier) [inline config/areas/site/dialogs.php:391]: Ha, that's clever!

**Review by bastianallgeier — APPROVED** [review]

**Pull request merged**